packages/hx-library/CHANGELOG.md

```
@@ -1,5 +1,313 @@
+  | `hx-split-button`  | `menuLabel`    | `labelMenu`    |
+  | `hx-overflow-menu` | `menuLabel`    | `labelMenu`    |
+
+  ### CSS Part Renames
+
+  | Component     | Old Part    | New Part       |
+  | ------------- | ----------- | -------------- |
+  | `hx-drawer`   | `close-btn` | `close-button` |
+  | `hx-carousel` | `prev-btn`  | `prev-button`  |
+  | `hx-carousel` | `next-btn`  | `next-button`  |
+
+  ## Non-Breaking Fixes
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)